Java/src/main/java/epam/Planes/experimentalPlane.java

```diff
@@ -0,0 +1,33 @@
+package epam.Planes;
+
+import epam.models.ClassificationLevel;
+
+
+public class experimentalPlane extends Plane {
+	private ClassificationLevel classificationLevel;
+
+	public experimentalPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity,
+			ClassificationLevel classificationLevel) {
+		super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
+		this.classificationLevel = classificationLevel;
+	}
+
+	public ClassificationLevel getClassificationLevel() {
+		return classificationLevel;
+	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
change the name of variable "o", variable names must be self descriptive

Reply:
Done